apiserver/billing/odoostorage/odoo/odoo8/odoo8.go

```diff
@@ -0,0 +1,168 @@
+package odoo8
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+
+	billingv1 "github.com/appuio/control-api/apis/billing/v1"
+	"github.com/appuio/control-api/apiserver/billing/odoostorage/odoo"
+	"github.com/appuio/control-api/apiserver/billing/odoostorage/odoo/odoo8/client"
+	"github.com/appuio/control-api/apiserver/billing/odoostorage/odoo/odoo8/client/model"
+)
+
+// Used to identify the accounting contact of a company.
+const roleAccountCategory = 7
```

Comment:
Is this generally valid for Odoo 8 or is this a value which is specific for the VSHN Odoo?

Reply:
I'm actually not sure and never cross checked. Setting up a Odoo8 instance could be a yak/

Reply:
From the not quite correct english in Odoo i'd guess it's VSHN custom. The whole logic with "Accounting" partners is very VSHN specific too.

Reply:
Odoo 16 seems to have a better handling of invoice addresses https://demo4.odoo.com/web#id=93&cids=1&menu_id=117&action=159&model=res.partner&view_type=form .

Where they are not just a tag.